tools/webhook_test.py

```diff
@@ -136,15 +139,18 @@ def set_init(webhook_type):
             "type": "raid",
             "message": {
                 "gym_id": 0,
-                "gym_name": "unknown",
+                "name": "Test gym",
+                "team": 1,
```

Comment:
Some of these changes are going to cause some unintended consequences. It might be best to just remove these changes all together - but I'll leave that up to @LegitDongo since the webhook tester is his baby.